### PR TITLE
Add random solar spawners

### PR DESCRIPTION
**Pull request by Radezolid**
## About the PR
I want to add a new spawner for solars.

## Why / Balance
More mapping freedom.

## Technical details
Added a new spawner.

## Media

image

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

None.

**Changelog**

no cl.

### CODE REVIEW

File: Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/solar.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  - type: Sprite
+    layers:
+      - sprite: DeltaV/Structures/Power/Generation/solar_panel.rsi
+        state: random_solar
+  - type: RandomSpawner
+    offset: 0
+    prototypes:
+      - SolarPanel
+      - SolarPanelBroken
+    rareChance: 0.65
```
Comment (IamVelcroboy) [inline Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/solar.yml]: replace this with `chance:` so there's occasionally nothing. Make it like 80%-85%  or so. Rarechance is used for a separate list of prototypes is defined.

File: Resources/Textures/DeltaV/Structures/Power/Generation/solar_panel.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+	"license":"CC-BY-SA-3.0",
+	"copyright":"Taken from https://github.com/discordia-space/CEV-Eris/blob/d1e0161af146835f4fb79d21a6200caa9cc842d0/icons/obj/power.dmi and modified. Modified by Radezolid for random solar spawner",
+	"size":{"x":32,"y":32},
+	"states":
+	[
+		{
+			"name": "random_solar"
+		}
+	]
+}
```
Comment (deltanedas) [inline Resources/Textures/DeltaV/Structures/Power/Generation/solar_panel.rsi/meta.json]: untroll all the json formatting so its like other files

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Review by IamVelcroboy — APPROVED** [review]

File: Resources/Textures/DeltaV/Structures/Power/Generation/solar_panel.rsi/meta.json (hunk 2 above)
Comment (IamVelcroboy) [reply Resources/Textures/DeltaV/Structures/Power/Generation/solar_panel.rsi/meta.json]: I guess, need DN to sign off on this

**Comment by IamVelcroboy** [comment]
@deltanedas are you satisfied with the meta file now?

**Review by Colin-Tel — APPROVED** [review]

**Review by deltanedas — APPROVED** [review]

**Pull request merged**